src/plugins/foxPage/hooks/useOtherOpportunities.ts

```diff
@@ -10,13 +10,16 @@ import { useFarmingApr } from 'plugins/foxPage/hooks/useFarmingApr'
 import { useLpApr } from 'plugins/foxPage/hooks/useLpApr'
 import { useMemo } from 'react'
 import { bnOrZero } from 'lib/bignumber/bignumber'
+import { selectFeatureFlags } from 'state/slices/preferencesSlice/selectors'
+import { useAppSelector } from 'state/store'
 
 import type { OpportunitiesBucket } from '../FoxCommon'
 import { OpportunityTypes } from '../FoxCommon'
 
 export const useOtherOpportunities = (assetId: AssetId) => {
-  const { farmingAprV4, isFarmingAprV4Loaded } = useFarmingApr()
-  const { lpApr, isLpAprLoaded } = useLpApr()
+  const featureFlags = useAppSelector(selectFeatureFlags)
```

Comment:
Destructure?

Reply:
Definitely, this naming makes more sense! Will tackle in a follow-up smallish PR so this can get in